### PR TITLE
implemented delegate for UIEventSubtype fixed #4 fixed #7

### LINKED ISSUE #4: Scroll duration

### LINKED ISSUE #7: RemoteControl assign
- [ ] becomeFirstResponder

### PR DESCRIPTION
**Pull request by SentulAsia**
fixed #4 and #7

### CODE REVIEW

**Comment by SentulAsia** [comment]
fixed #1 as well

**Comment by SentulAsia** [comment]
@ricardopereira I assume this is not what you want... care to enlightened me what should I remove?

File: ColiseuPlayer/ColiseuPlayer.swift [hunk 1]
```diff
@@ -230,9 +302,54 @@ public class ColiseuPlayer: NSObject
         }
     }
 
-    // isLastSong
+    public func isLastSong() -> Bool
+    {
+        if self.songsList != nil && self.currentSong != nil {
```
Comment (ricardopereira) [inline ColiseuPlayer/ColiseuPlayer.swift]: @SentulAsia Try to use `if let`,

Something like:
```
if let songs = self.songsList, let currentSong = self.currentSong { ...
```

**Comment by ricardopereira** [comment]
Really great work 🎉🎊
Thank you.

**Pull request merged**